src/socketio.quiver.js

```diff
@@ -115,23 +115,23 @@ QuiverSocialProvider.DEFAULT_SERVERS_ = [{
 }, {
   type: 'socketio',
   domain: 'dzgea1sj9ik08.cloudfront.net',
-  front: 'sdk.amazonaws.com'
+  front: 'js.arcgis.com'
 }, {
   type: 'socketio',
   domain: 'd2oi2yhmhpjpt7.cloudfront.net',
   front: 'api.mapbox.com'
 }, {
   type: 'socketio',
   domain: 'd2cqtpyb8m6x8r.cloudfront.net',
-  front: 'cdn.tinymce.com'
+  front: 'cdn.ckeditor.com'
 }, {
   type: 'socketio',
   domain: 'd3h805atiromvi.cloudfront.net',
   front: 'assets.tumblr.com'
 }, {
   type: 'socketio',
   domain: 'd2yp1zilrgqkqt.cloudfront.net',
-  front: 'www.splunk.com'
+  front: 'cdn.livefyre.com'
 }];
```

Comment:
Thanks!  Nope, I did this all "by hand", in _extremely_ labor-intensive fashion.  However, I think it would actually be quite straightforward to automate, and I think that would be very worthwhile, both for us and for other interested groups.